execution_workflows/APAtrap/modules/identify_distal_3utr.nf

```diff
@@ -0,0 +1,26 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+/*
+    Run the first step of APAtrap: identifyDistal3UTR to refine
+    annotated 3'UTRs and identify novel 3'UTRs or 3'UTR extensions.
+*/
+process IDENTIFY_DISTAL_3UTR {
+        publishDir "${params.outdir}/apatrap", mode: params.publish_dir_mode
+        container "docker.io/apaeval/apatrap:latest"
+
+        input:
+        tuple path(genome_file), path(reads_bedgraph_file)
+
+        output:
+        path "$utr_output", emit: ch_predictapa_input
+
+        script:
+        utr_output = "predictapa_input.bed"
+        """
+        identifyDistal3UTR -i $reads_bedgraph_file -m $genome_file -o $utr_output
```

Comment:
Looking at docs and source code there are a number of optional parameters for `identifyDistal3UTR`. I think these should be specified in pipeline (and set to default values in a config file) so we have the option to modify if we wish.

Optional parameters:
```
-w  window size used to scan the mapping result, default is 100 (has to be at least 20).
-e  pre-extension size of each 3'UTR, default is 10000 (has to be at least 100)
-c  minimum coverage of the end of the distal 3'UTR with comparing to the whole transcript, default is 0.05 (valid range 0-1).
-p  minimum percentage of valid nucleotides in a scanning-window, default is 0.8 (valid range 0-1).
-s  gene symbol file.
```
I don't think specifying gene symbol file is necessary though, but all the other optional parameters should be defined in my opinion.